bin/centos71-devbuild.sh

```diff
@@ -44,9 +44,9 @@ if [ "${1}" = "--fg" ]
 then
   echo "CentOS 7.1 Dev Build kicked off, Log being saved in ${LOG_FILE} ..."
 
-  time ruby /build/scripts/vmbuild.rb $BUILD_OPTIONS 2>&1 | tee ${LOG_FILE}
+  time ruby /build/manageiq/scripts/vmbuild.rb $BUILD_OPTIONS 2>&1 | tee ${LOG_FILE}
```

Comment:
I think removing the symlinks in the end is the best idea.  The symlinks are good in the interim when you are getting this working, so you can flip back and forth, but in the end they are just confusing noise.